chiselFrontend/src/main/scala/chisel3/core/package.scala

```diff
@@ -62,16 +62,16 @@ package object core {
 
   // These provide temporary compatibility for those who foolishly imported from chisel3.core
   @deprecated("Avoid importing from chisel3.core, these are not public APIs and may change at any time. " +
-    " Use chisel3.experimental.RawModule instead.", "since the beginning of time")
+    " Use chisel3.experimental.RawModule instead. This alias will be removed in 3.3.", "since the beginning of time")
```

Comment:
Are we still using experimental?

Reply:
Apparently, yes. A change there would likely be better in a separate PR, though?

Reply:
Huh, ok. Right, that was a milestone 3.3 item.